docs/outlook/append-on-send.md

```diff
@@ -229,12 +342,47 @@ For this scenario, you'll implement appending a disclaimer to the item when the
 
 1. Create a new message, and add yourself to the **To** line.
 
-1. From the ribbon or overflow menu, choose **Perform an action**.
+1. (Optional) Enter text in the body of the message.
+
+1. From the ribbon or overflow menu, select **Prepend header**.
+
+1. From the ribbon or overflow menu, select **Append disclaimer**.
+
+1. Send the message, then open it from your **Inbox** or **Sent Items** folder to view the inserted content.
+
+    ![A sample of a sent message with the Contoso header prepended and the disclaimer appended to its body.](../images/outlook-prepend-append-on-send.png)
+
+## Review feature behavior and limitations
+
+As you implement prepend-on-send and append-on-send in your add-in, keep the following in mind.
+
+- Prepend-on-send and append-on-send are only supported in compose mode.
+
+- The string to be prepended or appended must not exceed 5,000 characters.
+
+- HTML can't be prepended or appended to a plain text body of a message or appointment. Conversely, plain text can be added to an HTML-formatted body of a message or appointment.
+
+- Any formatting applied to prepended or appended content doesn't affect the style of the rest of the mail item's body.
+
+- Prepend-on-send and append-on-send can't be implemented in the same add-in that implements the [on-send feature](outlook-on-send-addins.md). As an alternative, consider implementing [Smart Alerts](smart-alerts-onmessagesend-walkthrough.md) instead.
+
+- When implementing Smart Alerts in the same add-in, the prepend-on-send and append-on-send operations occur before the `OnMessageSend` and `OnAppointmentSend` event handler operations.
+
+- If multiple active add-ins use prepend-on-send or append-on-send, the order of the content to be inserted depends on the order in which the add-in ran. For prepend-on-send, the content of the add-in that runs last appears at the top of the mail item above the previously prepended content. For append-on-send, the content of the add-in that runs last appears at the bottom of the mail item below the previously appended content.
+
+- Delegate and shared mailbox scenarios are supported as long as the add-in that implements prepend-on-send or append-on-send is enabled on the shared mailbox or delegator's account.
```

Comment:
When you say delegator, do you mean the mailbox owner or the authorized delegate? BTW, delegate and delegator have the same meaning.

Reply:
The mailbox owner. I changed it to say "owner" to avoid confusion.